css/src/atomics/colors.scss

```diff
@@ -69,6 +48,18 @@
 	}
 }
 
+@include tablet {
+	@each $name, $color-set in $colors {
+		$base: nth($color-set, $color-index-base);
+		$invert: nth($color-set, $color-index-invert);
+
+		.background-color-#{$name}-tablet {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+}
```

Comment:
After an additional investigation, it turned out there were no `desktop`/`widescreen` rules previously defined so removed those two loops.

But the `mobile` existed, so keeping the `tablet` instead to match the mobile-first approach. Not sure if I should though. Do we need to provide that much flexibility here? 🤔

@wibjorn @v-mingho @asrivastava0308

Reply:
I think providing tablet only is fine for now, given that we've done that with some other helpers (@asrivastava0308 can chime in on that). Tablet is the place where major layout shifts happen.